staramr/results/AMRDetectionSummaryResistance.py

```diff
@@ -1,6 +1,6 @@
 from collections import OrderedDict
 
-import numpy
+import numpy as np
```

Comment:
Looks like `numpy` is unused in this module.

Reply:
Thanks. I must have missed that d8202ef5eb312bf2a6d3d5800d5925cd37f87c06